labgrid/driver/networkusbstoragedriver.py

```diff
@@ -23,6 +26,21 @@ class NetworkUSBStorageDriver(Driver):
         },
     }
 
+    def _check_available(func): # pylint: disable=no-self-argument
+        """Check if a mass storage device is available as a decorator"""
+
+        @wraps(func)
+        def wrapper(self, *_args, **_kwargs):
```

Comment:
Just use a normal function here, the complexity of a decorator isn't work it for the two functions below (just call the check explicitly as the first step in the function).
Also, without the decorator, the backtraces are cleaner.